CONTRIBUTING.md

```diff
@@ -157,16 +160,49 @@ In the case that an objection is raised in a pull request by another collaborato
 
 ### When merging
 
+- All required Status-checks must have passed.
+- All discussions must be resolved.
 - [`squash`][] pull requests made up of multiple commits
 
-## Becoming a collaborator
+## Becoming a Collaborator
 
-- Collaborators must be actively contributing to the project
-- A Pull Request must be opened on the @nodejs/nodejs.org README file adding the new collaborator to the list (note the order of names)
-- The Pull Request must be approved by at least two members of @nodejs/website or @nodejs/tsc
-- The Pull Request must remain open for 72 hours without any objections
+A collaborator of the Node.js Website repository is a member of the Node.js Website Team.
 
-More details about Collaboration can be found in the [COLLABORATOR_GUIDE.md](./COLLABORATOR_GUIDE.md).
+The Website Team is responsible for the technical development of the Node.js Website, thus it is expected
+that team members have significant knowledge about modern Web Technologies and Web Standards.
+
+Note that regular contributors do not need to become "Collaborators". Any contribution is appreciated and a Collaborator status
+is a formality that comes with obligations.
+
+If you're an active contributor seeking to become a member we recommend reaching out to one of the existing Team Members for guidance.
+
+### What's the process for becoming a Collaborator?
```

Comment:
pointing out that this isn't in the table of contents, though other H3's are.

looking closer, there might be some heading to indent inconsistency that we can fix afterward

for example:

Developers CoA is an H2 but in the TOC is indented under Getting Started, also an H2